tools/alpine/Dockerfile

```diff
@@ -54,3 +54,5 @@ COPY --from=shellcheck /usr/local/bin/shellcheck /usr/local/bin/shellcheck
 COPY --from=shellcheck /usr/local/lib/ /usr/local/lib/
 
 RUN apk update && apk upgrade -a
+
+ARG CONTAINERD_COMMIT=bdf9f5f7388e8203e63a74b89800f7f3dd4a7743
```

Comment:
@justincormack what do you think of adding the commit SHA to alpine base. I kinda like it. and we could also add:
```
ENV DOCKER_BUCKET get.docker.com
ENV DOCKER_VERSION 17.05.0-ce
ENV DOCKER_SHA256 340e0b5a009ba70e1b644136b94d13824db0aeb52e09071410f35a95d94316d9
```
to it as we likely need this in a number of different packages

Alternatively we could add the source code and/or binaries to the alpine base